static/flow/views/landing-page-view.js

```diff
@@ -140,20 +167,22 @@ var LandingPageView = function(options) {
     //
     var createLandingPageMenuSection = function(name, content) {
         str = name.replace(/\s/g, '');
-        var id = "landing-page-" + str + "-diagrammenu";
-        div = $('<div>', {class: 'diagramMenu', id: id});
-        var header = $('<div>', {class: 'diagramMenuHeader concorddarkblue'} );
-        var title = $('<div>', {class: 'diagramMenuTitle noSelect'} ).text(name);
-        var chevron = $('<div>', {class: 'diagramMenuChevron glyphicon glyphicon-chevron-down', css:{color:'ffffff'}} );
+        str = str.toLowerCase();
+        var id = "landing-page-" + str + "-menu-section";
+        div = $('<div>', {class: 'menu-section', id: id});
+        var header = $('<div>', {class: 'menu-header container-dark-gray'} );
+        var title = $('<div>', {class: 'menu-title noSelect'} ).text(name);
+        var chevron = $('<div>', {class: 'diagram-menu-chevron noSelect'} );
+        var img = $('<img>'); //Equivalent: $(document.createElement('img'))
```

Comment:
I'd just pick the cleaner line of code here instead of including this comment

Reply:
Ah, that was a note to self when building.  Shouldn't be there.